lib/policy/config/expressionorlist.go

```diff
@@ -54,6 +54,9 @@ func (eol *ExpressionOrList) UnmarshalJSON(data []byte) error {
 }
 
 func (eol *ExpressionOrList) Valid() error {
+	if eol.Expression == "" && len(eol.All) == 0 && len(eol.Any) == 0 {
+		return ErrExpressionEmpty
+	}
```

Comment:
Good catch!

Reply:
Yeah I noticed that ErrExpressionEmpty was unused :) Had to go somewhere